### PR TITLE
Apexcore 293.migrate docs.v2

### PR DESCRIPTION
**Pull request by sashadt**

### CODE REVIEW

**Comment by sashadt** [comment]
image

**Comment by tweise** [comment]
@sashadt I have cleaned up this import and moved it to the following branch:

https://github.com/apache/incubator-apex-core/tree/APEXCORE-293/docs

Please double check. Also, if you find something in the remaining history that can/should be purged let me know.

I need to work on fixing the build failure next.

**Comment by tweise** [comment]
How will the deployment work with multiple versions?

File: docs/application_packages.md [hunk 1]
```diff
@@ -0,0 +1,667 @@
...
+that App Package file to launch this sample application in your actual
+Apex installation.
+
+## Using IDE
+
+Alternatively, you can do the above steps all within your IDE.  For
+example, in NetBeans, select File -\> New Project.  Then choose “Maven”
+and “Project from Archetype” in the dialog box, as shown.
+
+![](images/AppPackage/ApplicationPackages.html-image00.png)
```
Comment (tweise) [inline docs/application_packages.md:85]: @davidyan74 can we update the screen shot to reflect the changed groupId?

**Comment by sashadt** [comment]
Deployment is a separate issue which needs to be addressed.  The idea for now is that each version gets copied to a separate directory, and the main project website has links to each sub-directory for specific version.

**Pull request closed without merging**